script.js

```diff
@@ -0,0 +1,29 @@
+const menuBtn = document.querySelector('.menu');
+const menuList = document.querySelector('.navlinks');
+const menuItems = Array.from(document.querySelectorAll('.navlink-items'));
+const menuIcon = document.querySelector('.menu-icons');
+const mediaQuery = window.matchMedia('(min-width: 992px)');
+
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.toggle('active');
+  menuList.classList.toggle('active');
+
+  if (menuBtn.classList.contains('active')) {
+    menuIcon.src = './images/Disabled.svg';
+  } else {
+    // eslint-disable-next-line no-restricted-globals
+    location.reload();
```

Comment:
- [ ] [OPTIONAL] If you can find a way to avoid this eslint-disable comment, it will look be better 😉